[CN-1113]: Improve scaling and upgrading doc (#150)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/hazelcast-parameters.adoc

= Hazelcast System Properties

You can configure Hazelcast parameters using the `properties` field in the CRD spec. You can find all properties in link:https://docs.hazelcast.com/hazelcast/5.3/system-properties[System Properties] document.

It is worth highlighting the following system properties:

- During a rolling upgrade, the cluster is shutdown gracefully to prevent data loss. You can configure this using the link:https://docs.hazelcast.com/hazelcast/5.3/system-properties#hazelcast.graceful.shutdown.max.wait[hazelcast.graceful.shutdown.max.wait system property].
+
Example Configuration:
+
[source,yaml,subs="attributes+"]
----
include::ROOT:example$/properties.yaml[]
----

> Following system properties are set by Hazelcast Platform Operator to default values, and they cannot be changed:

Reviewer: ```suggestion
Hazelcast Platform Operator sets the following system properties to the default values, which cannot be changed:
```
